ext/phar/tests/phpinfo_004.phpt

```diff
@@ -25,7 +25,7 @@ phpinfo(INFO_MODULES);
 ===DONE===
 --EXPECTF--
 %a
-<h2><a name="module_phar">Phar</a></h2>
+<h2><a name="module_phar" href="#module_phar">Phar</a></h2>
```

Comment:
I've been itching to change it to `id` too, given the MDN recommendation and the more common usage of `id` for this. I was afraid this might be a little too much for a subtle change. If you think we can go ahead, I'd happily change `name` to `id`. Thank you.

Reply:
Well, maybe keep the `name` for now; we can still revise later (frankly, I don't think support for `name` is going away).